docs: changelog 2024-10-04 (#2247)
neondatabase/website: content/changelog/2024-10-04.md

---
description: tbd
---

### Sign up for Neon Deploy

Join us online on **October 30th** at **10:00 AM** PT to learn how Neon empowers developers to ship faster with Postgres. Tune in for Product updates and technical deep dives.

![join us at Neon Deploy](/docs/relnotes/neon-deploy.png)

### Neon on Azure is open for beta

Neon on Azure is now available for public beta! Create a [new project](https://console.neon.tech/app/projects/new?provider=azure) now and try it out.

![azure project creation globe](/docs/relnotes/azure_beta.png)

### More projects

> Software agents need room to grow, and we're dedicated to creating that space: we've expanded the project limits across all paid Neon plans.

Reviewer: ```suggestion
Customers with [database-per-tenant](https://neon.tech/use-cases/database-per-tenant), developer platforms, and AI Agents told us they needed more projects: we've expanded the project limits across all paid Neon plans.
```

### At a glance metrics for Free plan

We're making it easier for Free Plan users to check how much room you've got left on your account-level metrics. Find it on your Projects page.

![account metrics widget on all projects page](/docs/relnotes/account_metrics_widget.png)

And we've also added a project-level usage widget on the Project Dashbaord as well, letting you know your current consumption levels for the month for each individual project.

![project metrics on individual project page](/docs/relnotes/project_limits_widget.png)

### Guests are now Collaborators

We've renamed **Project sharing** to **Collaborators** in both your Project and Organization settings pages in the Neon Console. This update streamlines collaboration with other users, whether from a personal account project or with external users for Organization-owned projects.

To manage collaboration from the project level, **Project → Settings → Collaborators**. And to manage collaboration for all projects in your Organization, **Organization → People → Collaborators**.

<details>
<summary>**Fixes & improvements**</summary>

> - We've removed deprecated language and actions from the Neon CLI. A few months ago, we started calling your root branch `default` instead of `primary`. The CLI now reflects this change: `primary` is no more, it's `default` everywhere. For more info, see [set-default](/docs/reference/cli-branches#set-default).

Author: @bgrenon This one is not out yet. The PR is merged but a new CLI version has not been released.

Reviewer 2: Ok, i've got some related changes I need to pull out.

### Neon on Azure is open for beta

Neon on Azure is now available for public beta! Create a [new project](https://console.neon.tech/app/projects/new?provider=azure) now and try it out.

![azure project creation globe](/docs/relnotes/azure_beta.png)

### More projects

Customers with [database-per-tenant](https://neon.tech/use-cases/database-per-tenant), developer platforms, and AI Agents told us they needed more projects: we've expanded the project limits across all paid Neon plans.

New limits:

- **Launch Plan**: Up to 100 projects
- **Scale Plan**: Up to 1,000 projects
- **Business Plan**: Up to 5,000 projects

A reminder, last week we introduced the experimental [@neondatabase/toolkit](https://github.com/neondatabase/toolkit), which allows you to spin up a Postgres database in seconds and run SQL queries &#8212; ideal for quick setups and testing environments, and for taking advantage of your new, higher project limits.

### **ISO27110 & ISO27701 compliance**

These new certifications add to our growing list of compliance achievements. For more about Neon's compliance milestones, see [Compliance](/docs/security/compliance).

### Database limits per branch

> In case you missed last week's heads up, but this week we've started enforcing a limit of 500 roles and 500 databases per branch. If you have any concerns about these new limits, please reach out to the Neon Support team. Previously, these limits were recommended but not enforced.

Author: ```suggestion
In case you missed last week's heads up &#8212; this week we've started enforcing a limit of 500 roles and 500 databases per branch. If you have any concerns about these new limits, please reach out to the Neon Support team. Previously, these limits were recommended but not enforced.
```

> ### At a glance metrics for Free plan

Author: ```suggestion
### At a glance metrics for Free plan users
```
